workflows/sarek.nf

```diff
@@ -294,13 +291,17 @@ workflow SAREK {
     PREPARE_INTERVALS(fasta_fai)
 
     // Intervals for speed up preprocessing/variant calling by spread/gather
-    intervals_bed_combined        = (params.intervals && params.wes) ? Channel.fromPath(params.intervals).collect() : []
+    // this is not good, we need the combined bed for some tools that don't support scatter/gather. Why would we not use the same intervals for WGS?
+    // intervals_bed_combined        = (params.intervals && params.wes) ? Channel.fromPath(params.intervals).collect() : []
```

Comment:
If your alternative is working, can these comments then be removed?

Reply:
oh I almost missed this, no and yes, this warrants a separate PR, that i have already discussed with Oskar also in regard to haplotypecaller. I'll open a card